### PR TITLE
[11.0][MIG] partner_contact_department

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

Changed dependency.

### CODE REVIEW

File: partner_contact_department/README.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ To use this module, you need to:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/134/10.0
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
```
Comment (pedrobaeza) [inline partner_contact_department/README.rst]: README by fragments

**Review by pedrobaeza — APPROVED** [review]

File: partner_contact_department/README.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ Configuration
 
 To configure departments, you need to:
 
-* Go to *Sales > Configuration > Address Book > Departments*.
+* Go to *Contacts > Configuration > Address Book > Departments*.
```
Comment (LoisRForgeFlow) [inline partner_contact_department/README.rst]: `*Contacts > Configuration > Departments*`

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, minor comment

File: partner_contact_department/README.rst (hunk 2 above)
Comment (MiquelRForgeFlow) [reply partner_contact_department/README.rst]: done

**Comment by pedrobaeza** [comment]
Merging as trivial

**Pull request merged**